src/type-declarations/winston-logstash-transport.d.ts

```diff
@@ -1,14 +1,17 @@
-declare module 'winston-logstash-transport' {
+declare module 'logstash-tcp-wins' {
     import * as Transport from 'winston-transport';
 
     interface LogstashTransportOptions extends Transport.TransportStreamOptions {
         host?: string,
         port?: number,
+        json?: boolean,
         trailingLineFeed?: boolean,
         trailingLineFeedChar?: string
     }
 
-    export class LogstashTransport extends Transport {
+    class LogstashTransport extends Transport {
         constructor({}: LogstashTransportOptions)
     }
+
+    export = LogstashTransport;
```

Comment:
export the class in the same line you define it, like this:
`export class LogstashTransport extends Transport`

Reply:
it wont work this way, and i don't know another way except this to make it compile